openapi/customervoice/customervoice.yaml

```diff
@@ -109,6 +109,28 @@ components:
         status: proposed
       x-tags:
         - Review Requests
+      x-examples:
```

Comment:
FYI: If you define the example values on each field instead of on the request as a whole then new fields will show up automatically in the generated examples.

Reply:
I did that originally but it wasn't showing the id fields within the relationships data object. Not sure why.

Reply:
Did you mark them as readonly? That would cause the fields to be hidden in the generated sample request.